client/TracyProfiler.cpp

```diff
@@ -3965,6 +3965,74 @@ TRACY_API uint64_t ___tracy_alloc_srcloc_name( uint32_t line, const char* source
     return tracy::Profiler::AllocSourceLocation( line, source, sourceSz, function, functionSz, name, nameSz );
 }
 
+TRACY_API void ___tracy_emit_gpu_zone_begin_alloc( const struct ___tracy_gpu_zone_begin_data data, int active )
+{
+    if( !active ) return;
```

Comment:
These checks are not neccessary. User can simply not call the function in case of `active == false`.